src/pybind/mgr/rgw/module.py

```diff
@@ -181,6 +191,27 @@ def _cmd_rgw_realm_reconcile(self,
 
         return HandleCommandResult(retval=retval, stdout=out, stderr=err)
 
+    def serve(self) -> None:
+        self.run = True
+
+        self.log.info('Starting usage trim loop')
+        while self.run:
+            if self.usage_trim_older_than_days <= 0:
+                self.log.info('Skipping usage trim because usage_trim_older_than_days is not > 0')
+                self.event.sleep(self.usage_trim_interval)
+                continue
+            startDate = '1970-01-01'
+            endDate = (datetime.today() - timedelta(days=self.usage_trim_older_than_days)).strftime('%Y-%m-%d')
+            realms = cast(str, self.usage_trim_realms, default='').split(',')
+            for realm in realms:
+                self.log.info('Running usage trim for realm {} with startDate={} and endDate={}'.format(
+                              realm, startDate, endDate))
```

Comment:
Usage of f-strings is prefered for formatting: 

`f'Running usage trim for realm {realm} with startDate={startDate} and endDate={endDate}'`

Reply:
I will fix this!